packages/storybook-vue/stories/2_for_developers/02.ScaleAndVue_en.md

```diff
@@ -68,6 +68,15 @@ const app = Vue.createApp({});
 app.config.compilerOptions.isCustomElement = (tag) => tag.startsWith('scale-');
 ```
 
+If your project is using ESLint with eslint-plugin-vue, you should also deactivate the rule `vue/no-deprecated-slot-attribute`.
```

Comment:
Could you add the ticks to `eslint-plugin-vue` here too like in the DE version?

Reply:
Done :)